README.md

```diff
@@ -1,8 +1,18 @@
 # OracleMinimalApp-iOS
 Oracle Minimal App integration for iOS
-The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs. 
+The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs.
 To take the full advantage of the Oracle Cloud services, it is mandatory to integrate needed libraries/frameworks generally called as SDKs to your mobile app.
-At this moment, only iOS and Android mobile applications are supported by the SDK. 
+At this moment, only iOS and Android mobile applications are supported by the SDK.
 This page contains steps on how to update your mobile app’s source and integrate the app with Bluedot Point SDK.
 
 Please refer integration steps mentioned at : https://docs.bluedot.io/integrations/oracle-integration/
+
+For latest Bluedot Point SDK 15.6.1, New xcframework needs to be created as the PointSDK is built for iOS device and Simulators and it will report error as the OracleMinimalApp project only builds for iOS device.
```

Comment:
The issue here is not that this app is only built for iOS devices, but that the Carthage dependency has not been linked correctly. If we follow the [Carthage instructions in our docs](https://docs.bluedot.io/ios-sdk/ios-quick-start/) the app builds correctly.

Reply:
Carthage instructions are working, we don't need to rebuild the xcframework.
Removed the extra steps from readme